lighthouse-core/runner.js

```diff
@@ -212,6 +212,19 @@ class Runner {
       requestedUrl,
       settings: runnerOpts.config.settings,
     };
+
+    const internalSkipPageLoadForDevToolsA11y = Boolean(
+      global.isDevtools &&
```

Comment:
todo: convert this to an explicit parameter `__ internalSkipPageLoadForDevToolsA11y ` or similar that devtools can use to optionally audit the existing page as-is